luxonis_train/config/config.py

```diff
@@ -201,6 +201,32 @@ def check_graph(self) -> Self:
             raise ValueError("No outputs specified.")
         return self
 
+    @model_validator(mode="after")
+    def check_for_invalid_characters(self) -> Self:
+        for modules in [
+            self.nodes,
+            self.losses,
+            self.metrics,
+            self.visualizers,
+        ]:
+            invalid_char_error_message = (
+                "Name, alias or attached module contains a '/', which is not allowed. "
+                "Please rename the node or module to remove any '/' characters."
+            )
+            for module in modules:
```

Comment:
I think this is not entirely correct. The attached module checks should also check `alias` instead of just `name`. And I think it is enough in both cases to check just the name/alias.

```python
for module in modeles:
    if "/" in (module.alias or module.name):
        raise ...
```

`attached_to` must reference an existing alias so if the name there would contain slashes it will fail elsewhere that `attached_to` references a node that does not exist in the network.